clang/lib/Parse/ParseOpenMP.cpp

```diff
@@ -2974,10 +2974,8 @@ StmtResult Parser::ParseOpenMPDeclarativeOrExecutableDirective(
         Sema::CompoundScopeRAII Scope(Actions);
         AssociatedStmt = ParseStatement();
 
-        if (AssociatedStmt.isUsable() && isOpenMPLoopDirective(DKind) &&
-            getLangOpts().OpenMPIRBuilder)
-          AssociatedStmt =
-              Actions.OpenMP().ActOnOpenMPLoopnest(AssociatedStmt.get());
+        if (AssociatedStmt.isUsable() && isOpenMPLoopDirective(DKind))
+          AssociatedStmt = Actions.OpenMP().ActOnOpenMPLoopnest(AssociatedStmt.get());
```

Comment:
Can we just always call this now? Even if, do we need to? I mean, the only time you really need it is a standalone SIMD directive for target, right?